libsolidity/codegen/ExpressionCompiler.cpp

```diff
@@ -1694,7 +1692,7 @@ void ExpressionCompiler::appendExternalFunctionCall(
 	if (funKind == FunctionType::Kind::External || funKind == FunctionType::Kind::CallCode || funKind == FunctionType::Kind::DelegateCall)
 	{
 		m_context << Instruction::DUP1 << Instruction::EXTCODESIZE << Instruction::ISZERO;
-		m_context.appendConditionalInvalid();
+		m_context.appendConditionalRevert();
```

Comment:
Yes, the target contract could have selfdestructed, for example. If we use `Invalid` here, you cannot ever interact with contracts that might selfdestruct.